build/scripts/AppObjectValidation.psm1

```diff
@@ -150,6 +150,64 @@ function Get-FilesCollection
     }
 }
 
+<#
+    .SYNOPSIS
+    Tests that all application manifests in the specified path have the expected application versions, platform version and publisher name.
+    .DESCRIPTION
+    This function scans the specified path for app.json files, extracts the application and platform versions,
+    and checks if they match the expected values. If any discrepancies are found, an error is thrown.
+    .PARAMETER Path
+    The path to the source code directory to be scanned for app.json files.
+    .PARAMETER ExpectedAppVersion
+    The expected application version that should be present in the app manifests.
+    .PARAMETER ExpectedPlatformVersion
```

Comment:
This parameter doesn't seem to be used.

Reply:
Added a simple use of it now. It'll be hardcoded to the same major minor as the repoversion. In those few days where we might have a lower platform version while we're working on the version change, we can just hardcode it to something else